internal/target/repository.go

```diff
@@ -188,66 +198,31 @@ func (r *Repository) FetchAuthzProtectedEntitiesByScope(ctx context.Context, sco
 	return targetsMap, nil
 }
 
-// ListTargets in targets in a scope.  Supports the WithScopeId, WithLimit, WithType options.
+// ListTargets in targets in a scope.
```

Comment:
Can you update this go doc? What is currently here does not make sense to me. It should also note some about the impact of `listPermissions` on the results.

Reply:
Yep - Good point, added in https://github.com/hashicorp/boundary/pull/2319/commits/77810b9b59d88072eddef041e99a459218f4880a#diff-68b2bab8e324477709a446d45241a2dcf18c53519dcdad0718f97d0c30f6c161R216-R219